docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
```

Comment:
I can't believe this works

Reply:
@Rommelua I have tested it, it works. Could you tell me please, what is wrong with that version and which one should I use?